dom.bs

```diff
@@ -1478,7 +1478,8 @@ steps:
 <ol>
  <li>Let <var>signal</var> be this object's {{PromiseController/signal}}.
  <li>Set <var>signal</var>'s [=PromiseSignal/aborted flag=].
- <li>[=Fire an event=] named <code>abort</code> at <var>signal</var>.</li>
+ <li>[=Fire an event=] named <code>abort</code> at <var>signal</var>.
+ <li>Run <var>signal</var>'s [=PromiseSignal/abort steps=].
```

Comment:
I suspect we probably want the event to dispatch after the abort steps have run as the event listeners can cause side effects the abort steps might not anticipate.